posthog/migrations/0687_datamodelingjob.py

```diff
@@ -0,0 +1,57 @@
+# Generated by Django 4.2.18 on 2025-03-05 07:20
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import posthog.models.utils
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0686_alter_errortrackingissue_status"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="DataModelingJob",
+            fields=[
+                ("created_at", models.DateTimeField(auto_now_add=True)),
+                ("updated_at", models.DateTimeField(auto_now=True, null=True)),
+                (
+                    "id",
+                    models.UUIDField(
+                        default=posthog.models.utils.UUIDT, editable=False, primary_key=True, serialize=False
+                    ),
+                ),
+                (
+                    "status",
+                    models.CharField(
+                        choices=[("Running", "Running"), ("Completed", "Completed"), ("Failed", "Failed")],
+                        default="Running",
+                        max_length=400,
+                    ),
+                ),
+                ("rows_materialized", models.IntegerField(default=0)),
+                ("error", models.TextField(blank=True, null=True)),
+                ("workflow_id", models.CharField(blank=True, max_length=400, null=True)),
```

Comment:
I think we should record `workflow_run_id` too if the objective is to uniquely identify the workflow that executed a job.